lib/kitchen_sink/math.ex

```diff
@@ -24,6 +30,9 @@ defmodule KitchenSink.Math do
   iex> ceil(29.49, 10)
   30
   """
+  def ceil(_, 0) do
```

Comment:
the idea of significance seems non-standard
https://en.wikipedia.org/wiki/Floor_and_ceiling_functions

probably should rename this function